rapids-cmake/find/package.cmake

```diff
@@ -107,10 +114,14 @@ macro(rapids_find_package name)
   list(APPEND CMAKE_MESSAGE_CONTEXT "rapids.find.package")
   set(_rapids_options FIND_ARGS)
   set(_rapids_one_value BUILD_EXPORT_SET INSTALL_EXPORT_SET)
-  set(_rapids_multi_value GLOBAL_TARGETS)
+  set(_rapids_multi_value COMPONENTS GLOBAL_TARGETS)
   cmake_parse_arguments(_RAPIDS "${_rapids_options}" "${_rapids_one_value}"
                         "${_rapids_multi_value}" ${ARGN})
 
+  if(_RAPIDS_COMPONENTS)
+    list(APPEND _RAPIDS_UNPARSED_ARGUMENTS COMPONENTS ${_RAPIDS_COMPONENTS})
```

Comment:
Why don't we need to convert semicolons to spaces again here? Won't we have the same issue as in `rapids_cpm_find`?

Reply:
We don't have the same issue as the results getting directly written to a `find_package` command. The `rapids_cpm_find` does an extra level of argument parsing which treats the semi-colons as argument separators and therefore drops everything after the first component